sbncode/SBNEventWeight/Base/SmearingUtils.cxx

```diff
@@ -86,7 +86,7 @@ std::vector<std::vector<double> > MultiGaussianSmearing(
 std::vector<double> MultiGaussianSmearing(
     std::vector<double> const& centralValue,
     TMatrixD* const& inputCovarianceMatrix,
-    std::vector< double > rand) {
+    std::vector< float > rand) {
```

Comment:
It comes from the sbnobj:
https://github.com/SBNSoftware/sbnobj/blob/develop/sbnobj/Common/SBNEventWeight/EventWeightParameterSet.cxx#L64

It was changed because saving data in float can reduce the file size, where double has 8 bytes and float has 4 bytes.

Reply:
Have you studied if there is a difference between the results if you use floats or doubles? If not could we change these to doubles for now and then study if having floats change the answer?